[Core][AutoexplorePlus] MODWARN: nullable reference types

Had these warnings pop up in my `Player.log`:
```
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(7,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(10,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(12,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(13,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(16,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(17,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(18,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(19,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(20,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(23,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(24,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(25,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(28,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(29,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(32,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(33,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(36,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(38,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(41,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(42,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(43,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(44,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(45,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(46,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(49,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(50,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(53,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(54,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(55,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(58,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(59,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(62,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(65,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(66,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(67,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(70,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(71,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(72,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(73,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(74,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(75,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(76,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(80,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(81,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(82,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(85,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(86,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(87,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(88,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(91,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(92,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(95,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(98,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(99,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(103,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Data\QudSkillClasses.cs(106,24): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Utils\Output.cs(12,39): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Utils\Output.cs(21,34): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Core\Utils\Output.cs(35,36): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
MODWARN [Autoexplore+] - <...>\Mods\AutoexplorePlus\Main.cs(8,43): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
```

--- a/AutoexplorePlus/Main.cs
+++ b/AutoexplorePlus/Main.cs
@@ -1,4 +1,5 @@
-﻿using XRL;
+﻿#nullable enable
+using XRL;
 using XRL.World;
 using XRL.World.Parts.Mutation;
 

--- a/Core/Data/QudSkillClasses.cs
+++ b/Core/Data/QudSkillClasses.cs
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#nullable enable
+using System;
 
 namespace ModoMods.Core.Data {
   /// <summary>Known skill class names.</summary>

--- a/Core/Utils/Output.cs
+++ b/Core/Utils/Output.cs
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#nullable enable
+using System;
 using System.Globalization;
 using System.Reflection;
 using UnityEngine;

